cubic-server/Server.hpp

```diff
@@ -13,98 +13,50 @@
+#ifndef MC_PROTOCOL
+#define MC_PROTOCOL 759
+#endif
 
 class Server
 {
 public:
-    Server();
     ~Server();
 
     void launch();
 
+    const Configuration::ConfigHandler &getConfig() const {
```

Comment:
Should we have to put simple getters in hpp or should we put them in the corresponding cpp ?